core/controller/src/main/scala/whisk/core/controller/Meta.scala

```diff
@@ -217,13 +217,10 @@ trait WhiskMetaApi
     /** Store for identities. */
     protected val authStore: AuthStore
 
-    /** The route prefix e.g., /experimental. */
-    protected val routePath = "experimental"
```

Comment:
This is a problem if we want to support `v1/experimental/web` and `v2/web` simultaneously.